website/src/css/lib/prism.css

```diff
@@ -1,178 +1,121 @@
-/* http://prismjs.com/download.html?themes=prism&languages=markup+css+clike+javascript+bash+ruby+git+go+json+jsx+yaml&plugins=line-numbers */
```

Comment:
All changes in this file are the result of a copy paste of the [Tomorrow theme](https://github.com/PrismJS/prism/blob/gh-pages/themes/prism-tomorrow.css) to replace the default theme.